docs/UserGuide.md

```diff
@@ -176,27 +174,50 @@ Format: `edit [parameter/NEW_PARAMETER_DETAIL]...`
 * You can remove all the applicant’s tags by typing `t/` without specifying any tags after it.
 
 Examples:
-*  `edit 1 phone/91234567 email/bob@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `bob@example.com` respectively.
-*  `edit 2 name/Betty` Edits the name of the 2nd person to be `Betty`.
+*  `edit 1 p/91234567 e/bob@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `bob@example.com` respectively.
+*  `edit 2 n/Betty` Edits the name of the 2nd person to be `Betty`.
 
 
 ### 3.6 Locating applicants by field: `find`
 
-Finds applicants whose names contain any of the given keywords.
+Finds applicants whose has a field containing any of the given keywords among all specifed fields.
 
 Format: `find specifier/KEYWORD [more_specifier/MORE_KEYWORDS]`
 
+* At least one of the field specifier must be provided.
 * The search is case-insensitive. E.g., `bobby` will match `Bobby`.
 * The order of the keywords does not matter. E.g., `Bobby cortez` will match `Cortez bobby`.
-* Only full words will be matched e.g., `Bobby` will not match `Bobbys`.
-  
-  <!--- Can consider a flag for subwords in future increments) -->
-
-* Applicants matching at least one keyword will be returned (i.e., OR search). E.g., `Bobby Cortez` will return `Bobby Lacruz`, `Alexander Cortez`
+* If a field specifier is given more than once, the command will search only for the latest specifier. E.g., `n/Bobby n/cortez` will only search and return names with `cortez` in them.
+* Applicants matching at least one keyword will be returned (i.e., OR search). E.g., `n/Bobby Cortez` will return applicants with the name `Bobby Lacruz`, `Alexander Cortez`
+
+Search Types:
+1. Matching word search: Keywords will only match if there is a full matching word. E.g., `Bobby` will not match `Bobbys`.
+2. Substring Search: Keywords will match as long as a substring match exists. E.g., `Bob` will match `Bobby` and `Robobson`
+3. Numeric Search: Search will search by its numeric value. E.g., Both `3` and `3.0` will match `3.0`
```

Comment:
I think it's better to specify the search is for the CAP value not the max CAP, cause from here it's not so clear that we are checking only against the CAP value

Reply:
It's actually specified in the next paragraph, on what search type and on what value its used for.

Reply:
Yea, missed it. My bad